3506/3/.gitignore

```diff
@@ -0,0 +1 @@
+development.sqlite3
```

Comment:
It will exclude file from root directory, to exclude sqlite3 file do `db/development.sqlite3`